### ISSUE TITLE
Error when micro-showing images of type float

### ISSUE THREAD

**Issue by haesleinhuepf** [guiwitz/microfilm#10]
Hi @guiwitz ,

I'm hitting a little bug when trying to show two images of float type. When executing this code:

```
from skimage.data import cells3d
from microfilm.microplot import microshow
from skimage.filters import gaussian

data = cells3d()
membrane_image = data[30,1]

blurred = gaussian(image, sigma=5)

microshow([membrane_image, blurred])
```

I receive this error:
```
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
Input In [24], in <module>
      6 membrane_image = data[30,1]
      8 blurred = gaussian(image, sigma=5)
---> 10 microshow([membrane_image, blurred])

File ~\miniconda3\envs\bio_39\lib\site-packages\microfilm\microplot.py:107, in microshow(images, cmaps, flip_map, rescale_type, limits, num_colors, proj_type, channel_names, channel_label_show, channel_label_type, channel_label_size, scalebar_thickness, scalebar_unit_per_pix, scalebar_size_in_units, unit, scalebar_ypos, scalebar_color, scalebar_font_size, scalebar_text_centered, ax, fig_scaling, dpi, label_text, label_location, label_color, label_font_size, microim)
    103 #microim.images = colorify.check_input(microim.images)
    105 microim.rescale_type = colorify.check_rescale_type(microim.rescale_type, microim.limits)
--> 107 converted = colorify.multichannel_to_rgb(microim.images, cmaps=microim.cmaps, flip_map=microim.flip_map,
    108                                 rescale_type=microim.rescale_type, limits=microim.limits,
    109                                 num_colors=microim.num_colors, proj_type=microim.proj_type)
    111 if microim.cmaps is None:
    112         rgb = ['pure_red', 'pure_green', 'pure_blue']

File ~\miniconda3\envs\bio_39\lib\site-packages\microfilm\colorify.py:315, in multichannel_to_rgb(images, cmaps, flip_map, rescale_type, limits, num_colors, proj_type)
    311 if (limits is None) or (not any(isinstance(i, list) for i in limits)):
    312     limits = [limits for i in range(len(images))]
    314 converted = combine_image(
--> 315     [colorify_by_name(
    316         im, cmap_name=cmaps[ind],
    317         flip_map=flip_map[ind],
    318         rescale_type=rescale_type[ind],
    319         limits=limits[ind],
    320         num_colors=num_colors) for ind, im in enumerate(images)
    321     ], proj_type=proj_type)
    323 return converted

File ~\miniconda3\envs\bio_39\lib\site-packages\microfilm\colorify.py:315, in <listcomp>(.0)
    311 if (limits is None) or (not any(isinstance(i, list) for i in limits)):
    312     limits = [limits for i in range(len(images))]
    314 converted = combine_image(
--> 315     [colorify_by_name(
    316         im, cmap_name=cmaps[ind],
    317         flip_map=flip_map[ind],
    318         rescale_type=rescale_type[ind],
    319         limits=limits[ind],
    320         num_colors=num_colors) for ind, im in enumerate(images)
    321     ], proj_type=proj_type)
    323 return converted

File ~\miniconda3\envs\bio_39\lib\site-packages\microfilm\colorify.py:110, in colorify_by_name(image, cmap_name, flip_map, rescale_type, limits, num_colors)
     81 def colorify_by_name(image, cmap_name, flip_map=False, rescale_type='min_max', limits=None, num_colors=256):
     82     """
     83     Return 2D image as 3D RGB stack colored with a given colormap.
     84     
   (...)
    107 
    108     """
--> 110     image = rescale_image(image, rescale_type=rescale_type, limits=limits)
    112     cmap = cmaps_def(cmap_name, num_colors=num_colors, flip_map=flip_map)
    114     image_colored = cmap(image)

File ~\miniconda3\envs\bio_39\lib\site-packages\microfilm\colorify.py:184, in rescale_image(image, rescale_type, limits)
    162 """
    163 Rescale the image between 0-1 according to a rescaling type.
    164 
   (...)
    180     
    181 """
    183 if not np.issubdtype(image.dtype, np.unsignedinteger):
--> 184     raise Exception(f"Image should be unsigned integer but yours is {image.dtype}")   
    186 max_of_dtype = np.iinfo(image.dtype).max
    187 image = image.astype(np.float64)

Exception: Image should be unsigned integer but yours is float64
```

I'm wondering if it would be possible to also allow images of type float to be shown?

Thanks!
Robert

**Comment by guiwitz** [comment]
Hi @haesleinhuepf,

I actually fixed this in this commit 090aacc0245a90516be8bbf76d7a32ff5eddcbc9 but didn't do a new release in the meantime. I'll try to do one soon but in the meantime can you try to install with:
```
pip install git+https://github.com/guiwitz/microfilm.git
```
At least in my hands things work with float images on master.

**Comment by haesleinhuepf** [comment]
Confirmed, it works on current main.
